### DISCUSSION TITLE
More info on gererated images?

### DISCUSSION THREAD

**Discussion by rghedin** [wolfdevsllc/og-image-overlay#8] [Ideas]
This plugin is a little gem. Glad to have found it!

I was wondering if it would be possible to add more info on generated images, such as post title and post author, somewhat like what GitHub does:

![13476482525780707939](https://user-images.githubusercontent.com/19216581/227730921-e6797264-9cf9-432b-a8a7-97088eac8d28.png)

Is this feasible? Desirable?

**Comment by itsmereal** [comment]
@rghedin Thanks for trying out our plugin. We actually have this in our plans. Hopefully soon.
